### PR TITLE
updated gitignore, and corrected voting error with YesNoMaybe function

### PR DESCRIPTION
**Pull request by glen0071**

### CODE REVIEW

**Comment by jenessawhite** [comment]
also @glen0071 let's fix the spacing in this (yes I am that person lol)

File: src/components/results.js [hunk 1]
```diff
@@ -1,88 +1,96 @@
...
-	}
+    componentDidMount(props) {
+        this.user_can_vote();
+    }
 
 
-     //Method to determine if you can vote
+    //Method to determine if you can vote
 
     user_can_vote() {
```
Comment (jenessawhite) [inline src/components/results.js]: @DahirIlhan let's follow [javascript naming conventions](https://github.com/airbnb/javascript#naming-conventions) for this and use camel case. underscores are usually used in Ruby

**Comment by DahirIlhan** [comment]
@jenessawhite updated with the correct format for userCanVote. Thank you

**Comment by jenessawhite** [comment]
Looking good @DahirIlhan

**Pull request merged**